RELEASE-NOTES.md

```diff
@@ -1,6 +1,6 @@
 # Release Notes
 
-## PyMC3 3.10.0 (on deck)
+## PyMC3 3.10.0 (7 December 2020)
```

Comment:
You can add yourself as the release manager here. (As in the releases below.)

For the new "on deck" section I suggest something along the lines of `## PyMC3 latest (on deck)`.

Also, I think we forgot to mention & link some of the latest PRs in the release notes. This list is incomplete:
+ Model logp is now tested before starting to `find_map` or `sample` - surfacing inf/nan problems in the model earlier (see #4211)
+ `model_to_graphviz` now has a `formatting` kwarg that can be used to show more detailed node information (see #4260)

Reply:
I'm already listed as release manager (all the way at the bottom--it's a long list of changes ;) ), see #4268.

Checking model logp was mentioned already, but with #4116 (the issue) linked; I agree linking the PR (#4211) is better. I'll also add #4260 to the list of PRs related to the str representations. Other than that I think that's it?

Reply:
Oh yes I'll add the new "on deck" after creating the release, as per the [release guidelines](https://github.com/pymc-devs/pymc3/wiki/PyMC3-Release-Checklist).